core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -193,8 +216,21 @@ private[spark] object HighlyCompressedMapStatus {
     } else {
       0
     }
+    val threshold =
+      SparkEnv.get.conf.getLong("spark.shuffle.accurateBlkThreshold", 100 * 1024 * 1024) max
+        SparkEnv.get.conf.getInt("spark.shuffle.accurateBlkThresholdByTimesAvg", 2) * avgSize
```

Comment:
1. let's put the conf in `org.apache.spark.internal.config` object
2. I think we only need one conf, but not sure which one is better, cc @mridulm